charts/aws-ebs-csi-driver/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "0.10.1"
 name: aws-ebs-csi-driver
 description: A Helm chart for AWS EBS CSI Driver
-version: 0.10.3
+version: 0.10.4
```

Comment:
we can hold off on a helm release. it won't hurt, but it will also be pointless. Because the chart will include the new RBAC objects, but 0.10.1 of the driver doesnt need them yet.